### ISSUE TITLE
ImportError on macOS 11

### ISSUE THREAD

**Issue by ysBach** [pymc-devs/pymc#4679]
If you have questions about a specific use case, or you are not sure whether this is a bug or not, please post it to our discourse channel: https://discourse.pymc.io

## Description of your problem

**Please provide a minimal, self-contained, and reproducible example.**
```python
import pymc3 as pm
```

**Please provide the full traceback.**
```python
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-3-2e5c536b7c65> in <module>
----> 1 import pymc3 as pm

~/anaconda3/envs/2005ud2/lib/python3.8/site-packages/pymc3/__init__.py in <module>
      3 
      4 from .blocking import *
----> 5 from .distributions import *
      6 from .distributions import transforms
      7 from .glm import *

~/anaconda3/envs/2005ud2/lib/python3.8/site-packages/pymc3/distributions/__init__.py in <module>
----> 1 from . import timeseries
      2 from . import transforms
      3 from . import shape_utils
      4 
      5 from .continuous import Uniform

~/anaconda3/envs/2005ud2/lib/python3.8/site-packages/pymc3/distributions/timeseries.py in <module>
      4 
      5 from pymc3.util import get_variable_name
----> 6 from .continuous import get_tau_sigma, Normal, Flat
      7 from .shape_utils import to_tuple
      8 from . import multivariate

~/anaconda3/envs/2005ud2/lib/python3.8/site-packages/pymc3/distributions/continuous.py in <module>
     11 import warnings
     12 
---> 13 from pymc3.theanof import floatX
     14 from . import transforms
     15 from pymc3.util import get_variable_name

~/anaconda3/envs/2005ud2/lib/python3.8/site-packages/pymc3/theanof.py in <module>
      3 from theano import theano, scalar, tensor as tt
      4 from theano.configparser import change_flags
----> 5 from theano.gof import Op
      6 from theano.gof.graph import inputs
      7 from theano.sandbox.rng_mrg import MRG_RandomStreams

~/anaconda3/envs/2005ud2/lib/python3.8/site-packages/theano/gof/__init__.py in <module>
     38 from __future__ import absolute_import, print_function, division
     39 
---> 40 from theano.gof.cc import \
     41     CLinker, OpWiseCLinker, DualLinker, HideC
     42 

~/anaconda3/envs/2005ud2/lib/python3.8/site-packages/theano/gof/cc.py in <module>
     23 # gof imports
     24 from theano.gof import graph
---> 25 from theano.gof import link
     26 from theano.gof import utils
     27 from theano.gof import cmodule

~/anaconda3/envs/2005ud2/lib/python3.8/site-packages/theano/gof/link.py in <module>
     16 from theano.gof import utils
     17 from theano.gof import graph
---> 18 from theano.gof.type import Type
     19 
     20 from .utils import undef

~/anaconda3/envs/2005ud2/lib/python3.8/site-packages/theano/gof/type.py in <module>
     21 # Type #
     22 ########
---> 23 from theano.gof.op import CLinkerObject, Op
     24 
     25 __docformat__ = "restructuredtext en"

~/anaconda3/envs/2005ud2/lib/python3.8/site-packages/theano/gof/op.py in <module>
     23 from theano.gof import graph
     24 from theano.gof import utils
---> 25 from theano.gof.cmodule import GCC_compiler
     26 from theano.gof.fg import FunctionGraph
     27 

~/anaconda3/envs/2005ud2/lib/python3.8/site-packages/theano/gof/cmodule.py in <module>
     35 # we will abuse the lockfile mechanism when reading and writing the registry
     36 from theano.gof import compilelock
---> 37 from theano.configdefaults import gcc_version_str, local_bitwidth
     38 
     39 importlib = None

ImportError: cannot import name 'local_bitwidth' from 'theano.configdefaults' (/Users/ysbach/anaconda3/envs/2005ud2/lib/python3.8/site-packages/theano/configdefaults.py)
```

**Please provide any additional information below.**

## Versions and main components

What I did:
```
conda create --name 2005ud2 python=3.8 scipy=1.6 numpy=1.19 astropy=4.2 pandas=1.2 pymc3=3.8 theano-pymc mkl mkl-service jupyter corner -c conda-forge -y
conda activate 2005ud2
```
* PyMC3 Version:
* Aesara Version:
* Python Version:
* Operating system:
* How did you install PyMC3: (conda/pip)

**Comment by ColCarroll** [comment]
Hey! I can verify this is broken, but I am not sure we will fix it, since pymc3 v.3.8 is reasonably old. I got it to work by `pip uninstall pymc3 theano theano-pymc` and then `pip install -U pymc3==3.8 arviz==0.10`. Let me know if that helps!

**Comment by ysBach** [comment]
@ColCarroll 
Hi!
Thank you very much for the prompt and accurate reply.
After many trial and ``Error``s, I found conda is the problem for some reason.

I have finally succeeded in importing pymc3 after 

1. remove the env and make a new clean env with conda (``conda create --name test1 python=3.8``)
2. install needed packages with pip

This is very strange for me (fyi I am not even close to an expert when it comes to computer) but I am more than happy now as the problem is solved! 😄

Thanks for your help!
